server/models.py

```diff
@@ -526,6 +526,13 @@ def enroll_from_form(cid, form):
 
     @staticmethod
     @transaction
+    def unenroll(cid, user_id):
+        enrollment = Enrollment.query.filter_by(course_id=cid, user_id=user_id).one_or_none()
+        db.session.delete(enrollment)
```

Comment:
What happens when the enrollment doesn't exist? 

```
>>> db.session.delete(None)
 db.session.delete(None)
*** sqlalchemy.orm.exc.UnmappedInstanceError: Class 'builtins.NoneType' is not mapped
```

Maybe return None if the user isn't enrolled, otherwise return True